codesigndocuitests/codesigngroup.go

```diff
@@ -0,0 +1,424 @@
+package codesigndocuitests
+
+import (
+	"errors"
+	"fmt"
+	"path"
+	"sort"
+	"strings"
+
+	"github.com/bitrise-io/go-utils/colorstring"
+	"github.com/bitrise-io/go-utils/log"
+	"github.com/bitrise-io/goinp/goinp"
+	"github.com/bitrise-tools/codesigndoc/codesign"
+	"github.com/bitrise-tools/go-xcode/certificateutil"
+	"github.com/bitrise-tools/go-xcode/export"
+	"github.com/bitrise-tools/go-xcode/exportoptions"
+	"github.com/bitrise-tools/go-xcode/plistutil"
+	"github.com/bitrise-tools/go-xcode/profileutil"
+)
+
+// extractCertificatesAndProfiles returns the certificates and provisioning profiles of the given codesign group
+func extractCertificatesAndProfiles(codeSignGroups ...export.CodeSignGroup) ([]certificateutil.CertificateInfoModel, []profileutil.ProvisioningProfileInfoModel) {
+	certificateMap := map[string]certificateutil.CertificateInfoModel{}
+	profilesMap := map[string]profileutil.ProvisioningProfileInfoModel{}
+	for _, group := range codeSignGroups {
+		certificate := group.Certificate()
+		certificateMap[certificate.Serial] = certificate
+
+		installerCertificate := group.InstallerCertificate()
```

Comment:
installer certificate is used for macOS codesigning

Reply:
👍